adapters/acuityads/acuityads.go

```diff
@@ -0,0 +1,193 @@
+package acuityads
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"text/template"
+
+	"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/macros"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+type AcuityAdsAdapter struct {
+	endpoint template.Template
+}
+
+func NewAcuityAdsBidder(endpointTemplate string) *AcuityAdsAdapter {
+	template, err := template.New("endpointTemplate").Parse(endpointTemplate)
+	if err != nil {
+		return nil
+	}
+	return &AcuityAdsAdapter{endpoint: *template}
+}
+
+func GetHeaders(request *openrtb.BidRequest) *http.Header {
```

Comment:
Nitpick: Do we need this function to be exported? If it won't be used elsewhere, maybe we could `getHeaders(request *openrtb.BidRequest) *http.Header` instead

Reply:
You should also just return `http.Header` since the result is immediately dereferenced anyway.